cypress/integration/smoke.spec.ts

```diff
@@ -57,7 +57,8 @@ e2e.scenario({
               .type('{enter}');
             e2eSelectors.ConfigEditor.ClusterID.testID()
```

Comment:
you need to click first, wait for the clusters to load and then type here (see lines 113:115 for the example with secrets)